### PR TITLE
My first feature

### PR DESCRIPTION
**Pull request by julesvanrie**
First version of our feature.

### CODE REVIEW

**Comment by julesvanrie** [comment]
@otherjulesvanrie Can you review pls?

File: wip.py [hunk 1]
```diff
@@ -0,0 +1,3 @@
+print("Work in progresss")
```
Comment (otherjulesvanrie) [inline wip.py:1]: Do we still need this line?

**Review by otherjulesvanrie — CHANGES_REQUESTED** [review]

File: wip.py (hunk 1 above)
Comment (julesvanrie) [reply wip.py]: Indeed. Removed it.

**Review by otherjulesvanrie — APPROVED** [review]

**Comment by otherjulesvanrie** [comment]
LGTM

**Pull request merged**